Formula/parsnp.rb

```diff
@@ -41,6 +39,7 @@ def install
     # see: https://github.com/brewsci/homebrew-bio/pull/362
     cd "muscle" do
       ENV.deparallelize
+      system "./autogen.sh"
```

Comment:
Is this needed? What's the error without?

Reply:
Oh, I take it that it still includes `muscle`, but `muscle` no longer includes `./configure`?
Are you interested in trying to use `libmuscle` instead of the vendored dep?

Reply:
> Oh, I take it that it still includes `muscle`, but `muscle` no longer includes `./configure`?

Yeah

> Are you interested in trying to use `libmuscle` instead of the vendored dep?

I'm not sure; the comment right above this diff is cut off:

```rb
    # we still build this, but runtime will link against libmuscle
    # see: https://github.com/brewsci/homebrew-bio/pull/362
```

Reply:
Up to you. It's probably done this way just because that was easiest and didn't require modifying the build scripts.